nilearn/reporting/data/html/report_body_template_surfacemapsmasker.html

```diff
@@ -0,0 +1,108 @@
+{{inherit "base.html"}}
+<!--  -->
+{{def navbar}} {{enddef}}
+<!--  -->
+{{def report}}
+<div class="pure-g">
+  <div class="pure-u-1 pure-u-md-2-3">
+    <div class="image">
+      <script>
+        current_map_idx = 0;
+        displayed_maps = {{ displayed_maps }};
+        number_maps = {{ len(displayed_maps) }};
+      </script>
+      <div style="display: flex">
+        <div class="btn-container">
+          <button onclick="displayPreviousMap()" class="btn">
+            Previous Map
+          </button>
+        </div>
+
+        <div class="btn-container">
+          <h3 class="comp-title" style="margin: 0">
+            Region <span id="comp"></span>
+          </h3>
+        </div>
+
+        <div class="btn-container">
+          <button
+            onclick="displayNextMap()"
+            class="btn"
+            style="padding: 10px 20px; font-size: 16px"
+          >
+            Next Map
+          </button>
+        </div>
+      </div>
+
+      {{ figure|html }}
+    </div>
+  </div>
+  <div class="pure-u-1 pure-u-md-1-3">
+    {{if description}}
+    <p>{{description}}</p>
+    {{endif}}
+    <!--  -->
+    {{if number_of_regions}}
+    <p>
+      The masker has <b>{{number_of_regions}}</b> different potentially
+      overlapping regions.
+    </p>
+    {{if len(displayed_maps) != number_of_maps}}
+    <p>Only <b>{{len(displayed_maps)}}</b> regions are shown in this report.</p>
+    {{endif}}
+    <!--  -->
+    {{endif}}
+    <!--  -->
+    {{if n_vertices}}
+    <div>
+      <table class="pure-table">
+        <thead>
+          <tr>
+            <th>Hemisphere</th>
+            <th>Number of vertices</th>
+          </tr>
+        </thead>
+
+        <tbody>
+          {{py: vertices = n_vertices.items()}} {{for hemi, nb in vertices}}
+          <tr>
+            <td data-column="Hemisphere">{{hemi}}</td>
+            <td data-column="Number of vertices">{{nb}}</td>
+          </tr>
+          {{endfor}}
+        </tbody>
+      </table>
+    </div>
```

Comment:
realise that the glm report code has a way to easily turn pandas dataframe into html code: let's refactor after this PR is in